package CfgSignRewards;


message signDaliyReward{
	required int32 id   		= 1;
	required int32 days			= 2;
	required int32 item 		= 3;
	required int32 count 		= 4;
	required int32 vip_level 	= 5;
	required int32 times 		= 6;
	required int32 extra_item 	= 7;
}


message levelTargetRewards{
	required int32 career_level  = 1;
	required int32 level  		 = 2;
	required int32 limit_count   = 3;
	required int32 gift_id       = 4;
}

message SignRewards{
	repeated signDaliyReward signDaliyRewards = 1;
}

message LevelRewards{
	repeated levelTargetRewards levelReward = 1;
}